### PR TITLE
feat!: adding format column (image content type)

### PR DESCRIPTION
**Pull request by ronenkapelian**
Adding new Catalog entity of Format -> associated with the image content type.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,7 +40,8 @@
     "reflect-metadata": "^0.1.13"
   },
   "peerDependencies": {
-    "@map-colonies/mc-utils": ">=1.8.0"
+    "@map-colonies/mc-utils": ">=1.8.0",
+    "@map-colonies/types": "^1.3.0"
```
Comment (asafMasa) [inline package.json:44]: ">=1.3.0"

**Review by asafMasa — CHANGES_REQUESTED** [review]
👍

**Review by alebinson — APPROVED** [review]
looks good

File: package.json (hunk 1 above)
Comment (ronenkapelian) [reply package.json]: done

**Pull request merged**